scripts/generate-docs.js

```diff
@@ -0,0 +1,228 @@
+/**
+ * This script generates the SDK reference documentation based on the methods array defined at the beginning of the file.
+ * For a generated example, please refer to https://avaprotocol.org/docs/ethereum/develop/js-sdk
```

Comment:
hmm dead link

Reply:
The link was on staging website.
https://oak-website-git-staging-avaprotocol.vercel.app/docs/ethereum/develop/js-sdk

I’m deploying it to main